governance/polls/templates/Add $ilk to Liquidations 2.0 Framework - $date_MONTH_DD,_YYYY.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Add $ilk to Liquidations 2.0 Framework - $date_MONTH_DD,_YYYY
+summary: This poll proposes system parameters which could be used to initialize $ilk with the Liquidations 2.0 Framework.
+discussion_link: $risk_link
+vote_type: Plurality Voting
+categories:
+   - Collateral
+   - Risk Variable
+options:
+   0: Abstain
+   1: Yes
+   2: No
+---
+# Poll: Add $ilk to Liquidations 2.0 Framework - $date_MONTH_DD,_YYYY
+
+The Governance Facilitators have placed a Governance Poll into the [voting system](https://vote.makerdao.com/polling) on behalf of the MakerDAO mandated actors. This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on $date_DAY,_MONTH_DD at 16:00 UTC.
+
+**This is a binary vote. In a binary vote, you may vote for a single option. You should vote for the option which you prefer. Alternatively, if you would accept either of the two options and have no preference between them, you should abstain.**
+
+## Review
+
+This poll allows the MakerDAO governance community to signal their support or opposition to adding $ilk to the Liquidations 2.0 Framework in the Maker Protocol with the parameters below:
+
+* Underlying Collateral: $collateral_asset
+* [Price Change Multiplier]($cut_link): $price_change_multiplier
+* [Price Change Interval]($step_link): $price_change_interval
+* [Auction Price Multiplier]($buf_link): $auction_price_multiplier
+* [Proportional Kick Incentive]($chip_link): $proportional_kick_incentive
+* [Flat Kick Incentive]($tip_link): $flat_kick_incentive
+* [Maximum Auction Drawdown]($cusp_link): $maximum_auction_drawdown
+* [Maximum Auction Duration]($tail_link): $maximum_auction_duration
+* [Local Liquidation Limit]($ilk_hole_link): $local_liquidation_limit
+* [Liquidation Penalty]($chop_link): $liquidation_penalty
+* [Global Liquidation Limit]($hole_link): $global_liquidation_limit
```

Comment:
Global limit isn't a part of the ilk-specific parameters.

Reply:
We should add it manually for the first poll, but it shouldn't be in the template.